src/Microsoft.ServiceHub.Framework/Microsoft.ServiceHub.Framework.csproj

```diff
@@ -35,6 +35,9 @@
       <Generator>ResXFileCodeGenerator</Generator>
     </EmbeddedResource>
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="Extensions\" />
+  </ItemGroup>
```

Comment:
Can we revert this? We don't need it except for empty folders that we still want to appear in the project tree.

Reply:
Done.